lib/modules/manager/nix/readme.md

```diff
@@ -0,0 +1,8 @@
+The [`nix`][1] manager supports:
+
+- [`lockFileMaintenance`][2] updates for `flake.lock`
+- Extracts [nixpkgs][3] references
+
+[1]: https://github.com/NixOS/nix
+[2]: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance
+[3]: https://github.com/NixOS/nixpkgs
```

Comment:
```suggestion
The `nix`[^1] manager supports:

- `lockFileMaintenance` [^2] updates for `flake.lock`
- Extracts `nixpkgs` [^3] references

[^1]: [NixOS docs on GitHub](https://github.com/NixOS/nix)
[^2]: [Renovate docs for `lockfileMaintenance`](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance)
[^3]: [NixOS docs for `nixpkgs`](https://github.com/NixOS/nixpkgs)
```

### Correct syntax

My suggestion has the correct syntax for Material for MkDocs footnotes, key differences:

- You need to use a `^` before the number
- You must use `[]()` for the links to work
- Might as well use human-readable link names while we're at it 😉 

### How to enable footnotes on our docs site

If we decide we want to use footnotes in our docs we must enable the `footnotes` Markdown extension in the `mkdocs.yml` config file on the docs repository. See PR:

- https://github.com/renovatebot/renovatebot.github.io/pull/248

### Preview of footnotes

![preview-footnotes-in-docs-site](https://user-images.githubusercontent.com/34918129/200274238-c3e1ccdb-19e8-4c03-88b0-139d89d3edff.png)

Reply:
@HonkingGoose Those links aren't footnotes. Those are id links. so in text you refetence a id with `[text][id]` and then you put the id list at any place of the file as

```md
[id]: link "title"
``` 

https://commonmark.org/help/tutorial/07-links.html

Reply:
> ### "Title" link style Markdown code
> 
> ```markdown
> The `nix`[1] manager supports:
> 
> - `lockFileMaintenance` [2] updates for `flake.lock`
> - Extracts `nixpkgs` [3] references
> 
> [1]: https://github.com/NixOS/nix "NixOS docs on GitHub"
> [2]: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance "Renovate docs for `lockfileMaintenance`"
> [3]: https://github.com/NixOS/nixpkgs "NixOS docs for `nixpkgs`"
> ```
> 
> ### Preview of new syntax
> 
> Here's the preview of the Markdown code from above. I'm "hovering my mouse" over number 1 in this screenshot. The footnotes are _not_ listed on the bottom when I use @viceice's syntax:
> 
> ![new-syntax-preview](https://user-images.githubusercontent.com/34918129/200282280-5f4da11e-e930-4287-96c4-a32dff4c5533.png)

your link syntax is wrong. you missed the brackets on the keyword, which should be the link later and point to the url from the id.

Reply:
**so this**

```md
The [`nix`][1] manager supports:

- [`lockFileMaintenance`][2] updates for `flake.lock`
- Extracts [nixpkgs][3] references

[1]: https://github.com/NixOS/nix
[2]: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance
[3]: https://github.com/NixOS/nixpkgs
```

**renders as this on github**

The [`nix`][1] manager supports:

- [`lockFileMaintenance`][2] updates for `flake.lock`
- Extracts [nixpkgs][3] references

[1]: https://github.com/NixOS/nix
[2]: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance
[3]: https://github.com/NixOS/nixpkgs

Reply:
> your link syntax is wrong. you missed the brackets on the keyword, which should be the link later and point to the url from the id.

Oh, okay! 🙈😄 

I've copy/pasted your code and put it in my Material for MkDocs preview:

### Markdown code

```markdown
The [`nix`][1] manager supports:

- [`lockFileMaintenance`][2] updates for `flake.lock`
- Extracts [nixpkgs][3] references

[1]: https://github.com/NixOS/nix
[2]: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance
[3]: https://github.com/NixOS/nixpkgs
```

### Material for MkDocs preview

There's no footnotes section at the bottom of the page in the preview:

![newer-syntax-preview](https://user-images.githubusercontent.com/34918129/200295854-90385997-2731-4af0-9e96-43eb24553520.png)

Reply:
so works as expected. maybe you where confused by the number id's (which are not footnotes here), those can be arbitrary strings. 🙃

Reply:
> so works as expected. maybe you were confused by the number id's (which are not footnotes here), those can be arbitrary strings. 🙃

Yeah, this syntax is confusing me. I was really expecting to get footnotes. 😄 

I'm voting "use normal links", so like this:

```suggestion
The [`nix`](https://github.com/NixOS/nix) manager supports:

- [`lockFileMaintenance`](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance) updates for `flake.lock`
- Extracts [nixpkgs](https://github.com/NixOS/nixpkgs) references
```

Reply:
Sorry for the confusion! I've gotten into the habit of using number ids to not break up the flow of text when writing longer markdown. I can convert these back to regular links.